src/python_tools/tftp_transfer/host.py

```diff
@@ -0,0 +1,100 @@
+import tftpy, hashlib, os, argparse, pathlib, shutil
+
+TEMP = os.path.join(os.getcwd(), ".temp")
```

Comment:
you can actually use the `tempfile` library here, it will create a tempfile for you that is managed until the file loses scope